### PR TITLE
add checkbox scene

### PR DESCRIPTION
**Pull request by annieyw**
First draft:
checkbox scene

### CODE REVIEW

**Comment by mmalerba** [comment]
Can you get rid of the disabled checkbox? It doesn't look great. Also, is all of the text the same color here? I feel like every time I focus on one of them the others seem to look different, maybe it's an optical illusion...

**Comment by annieyw** [comment]
image

I think the disabled checkbox text _might_ have been a different colour because now that it's gone they look the same.

File: scenes/src/app/scenes/checkbox/checkbox-scene.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.scene-checkboxes {
+  width: 120px;
+  float: left;
```
Comment (mmalerba) [inline scenes/src/app/scenes/checkbox/checkbox-scene.scss:3]: can you make the host `display: flex` instead of using `float: left`

**Comment by annieyw** [comment]
image

**Comment by mmalerba** [comment]
@annieyw needs rebase

**Pull request merged**